### ISSUE TITLE
More than one chart in the same page

### ISSUE THREAD

**Issue by aner87** [tvjsx/trading-vue-js#199]
Hello, 
I am a new trading-vue-js adopter and I have encountered one thing I can't figure out for the life of me:
whenever I try to put more than one chart in the same page, the two charts don't stay independent, but they display the same data, even though I assign charts.data independently... I even tried physically separating the 2 charts in 2 different components, but I still face the issue.
The second chart also appears less sharp (in definition) than the first one, but I suppose that's just an issue due to the fact that I am doing something wrong with the data.

TL;DR
What is the best practice to display 2 (or more) completely data independent charts in the same page?

**Comment by C451** [comment]
Theres an example if you run `npm run test`

**Comment by aner87** [comment]
Thanks, for some reason I was able to make the example work, but now with a DataCube data structure.
Now I figure it out.
Thank you very much
